gradle/buildSrc/src/main/kotlin/io/spine/gradle/internal/deps.kt

```diff
@@ -0,0 +1,303 @@
+    )
+    val errorProneCheckApi     = "com.google.errorprone:error_prone_check_api:${Versions.errorProne}"
+    val errorProneCore         = "com.google.errorprone:error_prone_core:${Versions.errorProne}"
+    val errorProneTestHelpers  = "com.google.errorprone:error_prone_test_helpers:${Versions.errorProne}"
+    val checkerAnnotations     = "org.checkerframework:checker-qual:${Versions.checkerFramework}"
+    val checkerDataflow        = listOf(
+            "org.checkerframework:dataflow:${Versions.checkerFramework}",
+            "org.checkerframework:javacutil:${Versions.checkerFramework}"
+    )
+    val autoCommon             = "com.google.auto:auto-common:${Versions.autoCommon}"
+    val autoService = AutoService
+    val jsr305Annotations      = "com.google.code.findbugs:jsr305:${Versions.findBugs}"
+    val guava                  = "com.google.guava:guava:${Versions.guava}"
+    val flogger                = "com.google.flogger:flogger:${Versions.flogger}"
+    val slf4j                  = "org.slf4j:slf4j-api:${Versions.slf4j}"
+    val protobuf = listOf(
+            "com.google.protobuf:protobuf-java:${Versions.protobuf}",
+            "com.google.protobuf:protobuf-java-util:${Versions.protobuf}"
+    )
+    val protoc                 = "com.google.protobuf:protoc:${Versions.protobuf}"
+    val googleHttpClient       = "com.google.http-client:google-http-client:${Versions.httpClient}"
+    val googleHttpClientApache = "com.google.http-client:google-http-client-apache:${Versions.apacheHttpClient}"
+    val appengineApi           = "com.google.appengine:appengine-api-1.0-sdk:${Versions.appengineApi}"
+    val firebaseAdmin          = "com.google.firebase:firebase-admin:${Versions.firebaseAdmin}"
+    val jacksonDatabind        = "com.fasterxml.jackson.core:jackson-databind:${Versions.jackson}"
+    val roasterApi             = "org.jboss.forge.roaster:roaster-api:${Versions.roaster}"
+    val roasterJdt             = "org.jboss.forge.roaster:roaster-jdt:${Versions.roaster}"
+    val animalSniffer          = "org.codehaus.mojo:animal-sniffer-annotations:${Versions.animalSniffer}"
+    val ci = "true".equals(System.getenv("CI"))
+    val gradlePlugins = GradlePlugins
+}
+
+object Gen {
+    val javaPoet = "com.squareup:javapoet:${Versions.javaPoet}"
+}
+
+object Grpc {
+    val grpcCore               = "io.grpc:grpc-core:${Versions.grpc}"
+    val grpcStub               = "io.grpc:grpc-stub:${Versions.grpc}"
+    val grpcOkHttp             = "io.grpc:grpc-okhttp:${Versions.grpc}"
+    val grpcProtobuf           = "io.grpc:grpc-protobuf:${Versions.grpc}"
+    val grpcNetty              = "io.grpc:grpc-netty:${Versions.grpc}"
+    val grpcNettyShaded        = "io.grpc:grpc-netty-shaded:${Versions.grpc}"
+    val grpcContext            = "io.grpc:grpc-context:${Versions.grpc}"
+}
+ 
+object Runtime {
+    val floggerSystemBackend = "com.google.flogger:flogger-system-backend:${Versions.flogger}"
```

Comment:
Can we have nested nested objects? So that it's `Runtime.Flogger.systemBackend`?